tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,70 @@
+import ast
+
+import requests
+import yaml
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def getting_ceph_version(link):
```

Comment:
Change `link` to `url`

Reply:
Thanks @vamahaja
updated variable as "url".